src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -128,6 +137,16 @@ function RequestedAppointmentDetailsPage({
           />
         )}
 
+      {isCC &&
+        isCCRequest && (
+          <>
+            <h2 className="vaos-appts__block-label vads-u-margin-bottom--0 vads-u-margin-top--2">
+              Preferred community care provider
+            </h2>
+            <span>{practitionerName}</span>
+          </>
+        )}
+
```

Comment:
Should we be showing this "Preferred community care provider" header if there is no provider/we have no practitioner name? May want to check with @outerpress

Reply:
@narin @vbahinwillit - good catch, there shouldn't be a header if there's no practitioner name.

Reply:
I was wondering about that.